src/renderers/dom/shared/ReactDOMDebugTool.js

```diff
@@ -62,6 +62,27 @@ var ReactDOMDebugTool = {
   onTestEvent() {
     emitEvent('onTestEvent');
   },
+  onBeginProcessingChildContext() {
+    emitEvent('onBeginProcessingChildContext');
+  },
+  onEndProcessingChildContext() {
+    emitEvent('onEndProcessingChildContext');
+  },
+  onSetState() {
+    emitEvent('onSetState');
+  },
+  onMountRootComponent(internalInstance) {
```

Comment:
Yes,  have rebased
On Apr 26, 2016 6:45 PM, "Dan Abramov" <notifications@github.com> wrote:

> In src/renderers/dom/shared/ReactDOMDebugTool.js
> <https://github.com/facebook/react/pull/6398#discussion_r61183415>:
>
> > @@ -62,6 +62,27 @@ var ReactDOMDebugTool = {
> >    onTestEvent() {
> >      emitEvent('onTestEvent');
> >    },
> > +  onBeginProcessingChildContext() {
> > +    emitEvent('onBeginProcessingChildContext');
> > +  },
> > +  onEndProcessingChildContext() {
> > +    emitEvent('onEndProcessingChildContext');
> > +  },
> > +  onSetState() {
> > +    emitEvent('onSetState');
> > +  },
> > +  onMountRootComponent(internalInstance) {
>
> Now that #6549 <https://github.com/facebook/react/pull/6549> landed, the
> argument is debugID rather than internalInstance.
> Would you mind rebasing on that?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/react/pull/6398/files/ba8e02edad10495fb868586ec53519b881fa4f3e#r61183415>
>